uw_spotseeker/dao.py

```diff
@@ -25,26 +31,92 @@ def _get_live_implementation(self):
 
 
 class Spotseeker_LiveDAO(LiveDAO):
-    def load(self, method, url, headers, body):
+    # decrease cache expiry by epsilon seconds to account for time it takes to
+    # get token and set in cache, and time it takes to make request once
+    # retrieved from cache
+    EPSILON = 60
+
+    def load(self, method, url, headers, body) -> MockHTTP:
         if body is None:
             body = ''
-        body = body.encode("utf-8")
+        logger.debug(f'body: {body}')
 
-        consumer = oauth2.Consumer(key=settings.SPOTSEEKER_OAUTH_KEY,
-                                   secret=settings.SPOTSEEKER_OAUTH_SECRET)
-        client = oauth2.Client(consumer)
         url = settings.RESTCLIENTS_SPOTSEEKER_HOST + url
 
-        resp, content = client.request(url,
-                                       method=method,
-                                       body=body,
-                                       headers=headers)
-        response = self.process_response(resp, content)
+        headers['Authorization'] = self.get_auth_header()
+
+        if 'files' in headers:
+            files = headers['files']
+            del headers['files']
+        else:
+            files = None
```

Comment:
way to get the images to get posted through requests, seems to be the only way (AKA files=...)